src/pages/projects.js

```diff
@@ -422,6 +422,35 @@ const ProjectDescriptions = () => (
 
     <TitleWithAnchor headerClassName="projects-title projects-common-title">Emerging Projects</TitleWithAnchor>
 
+    <div className="project-box">
+      <TitleWithAnchor headerClassName="project-major-title projects-common-title" headerTag="h3">eBPF for Windows</TitleWithAnchor>
+      <div className="project-minor-title">
+        eBPF Runtime for Windows
+      </div>
+      <div className="project-body">
+        <!-- no logo exists yet -->
```

Comment:
Netlify complains about this comment. I'm not very familiar with the framework in use, but my guess is that even if we use some HTML tags the current file is JavaScript, and comments would be `//` or `/* */`? No idea how to _generate_ HTML comments (but then we probably just want a comment in the file anyway, not in the generated site).

Reply:
Removed comment for now